components/connector/before-commit.sh

```diff
@@ -55,9 +55,9 @@ if [[ "$1" == "$CI_FLAG" ]]; then
 	buildEnv="env CGO_ENABLED=0 GOOS=linux GOARCH=amd64"
 fi
 
-${buildEnv} go build -o connector ./cmd/main.go
+${buildEnv} go build -o bin/connector ./cmd/main.go
```

Comment:
don't mix many fixes in one PR, this is extremely important PR and you decided to fix storing binaries in our project...

Reply:
Because I accidentally commited one binary in previous PR, so that's why I fixed it "in the meantime".